Documentation/KnownIssues/776516-InvalidOperationException in System_Web_ni!System.Web.Hosting.RecycleLimitMonitor+RecycleLimitMonitorSingleton.AlertProxyMonitors.md

```diff
@@ -0,0 +1,25 @@
+# InvalidOperationException in ASP.Net-RecycleLimitMonitor-AlertProxyMonitors
+
+## Symptoms
+ASP.Net applications may crash intermittently due to an `InvalidOperationException` in
+`System_Web_ni!System.Web.Hosting.RecycleLimitMonitor+RecycleLimitMonitorSingleton.AlertProxyMonitors()`.
+
+## Cause
+This issue was introduced when ASP.Net 4.7 separated the concerns of cache management and memory monitoring. This new
+`RecycleLimitMonitorSingleton` monitors worker process private bytes usage against the recycling limit for the process, and when
+the limit is near, it alerts a set of "proxy monitors" in each registered AppDomain so they can take action to avoid a process
+recycle. The collection of "proxy monitors" being enumerated was not thread-safe though, so new AppDomains spinning up or old
+AppDomains shutting down while this monitor enumerated the collection could result in this unhandled `InvalidOperationException`
+in the default AppDomain, causing a crash.
+
+## Workaround 
+There is no reliable workaround. If the environment does not tend to consume private bytes near the recycling limit, or if
+web app domain lifetimes tend to last the entire length of the process lifetime, then chances of hitting this issue are
+extremely minimal. More private bytes consumption and/or more AppDomain recycling will increase the odds of hitting this.
+
+## Resolution
+A hotfix for this issue is in the works for .Net 4.8. This page will be updated with a link to the hotfix when it becomes
```

Comment:
```suggestion
A hotfix for this issue is planned for .Net 4.8. This page will be updated with a link to the hotfix when it becomes
```

Reply:
@rpetrusha can you please give me the link to the net48 branch?  

I was confused when I created this PR, this is the link from the email I received.

Reply:
If you're working with bash, you have to check out the **net48** branch, then create a new branch that based on it. When you open the PR in GitHub, target the **net48** branch, as shown in this image:
![image](https://user-images.githubusercontent.com/10886961/55441438-bf39c880-555f-11e9-8a71-d4f8f5ac9ea6.png)
Let me know if that doesn't help or you're still having difficulty, @HongGit.